composer.json

```diff
@@ -23,6 +23,7 @@
     ],
     "require": {
         "php": "^8.1",
+        "composer-runtime-api": "^2.2",
```

Comment:
I think we should use `^2.0` here - people may be using a version of Composer that’s between 2.0.0 and 2.2.0, and it can be hard(er) to upgrade infrastructure/build pipeline packages. I don’t think we’re using anything that wasn’t available in 2.0.0

Reply:
done